src/libs/antares/study/parts/thermal/prepro.cpp

```diff
@@ -294,7 +294,7 @@ void PreproThermal::reset()
     }
 }
 
-bool PreproThermal::normalizeAndCheckNPO(const AnyString& clustername, uint clusterSize)
+bool PreproThermal::normalizeAndCheckNPO(const AnyString& clustername, uint clusterSize, const AnyString& areaname)
```

Comment:
I recommend re-ordering the arguments of this function:

1. Area name
2. Cluster name
3. Cluster size

Reply:
Done !